### PR TITLE
Populate LaunchProfileValue.DisplayName

### PR DESCRIPTION
**Pull request by tmeschter**
Move to a newer version of CPS that includes the new `LaunchProfileValue.DisplayName` property and populate it. For the moment it will be the same as the `Name` property, but a future change will make `Name` a unique, persistent identifier for a launch profile.

Note this is the second step of a set of staged changes. CPS now defines the `LaunchProfileValue.DisplayName` property but can't take a dependency on it because the .NET Project System doesn't populate it at all. And since CPS can't depend on it, the project system can't be updated to make `Name` a persistent, unique value and handle renames as expected. This interim change allows CPS to move forward without a user-visible break or change in behavior.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/project-system/pull/7811)

### CODE REVIEW

**Comment by tmeschter** [comment]
This will remain a draft until we can pick up the real versions of the CPS packages with `LaunchProfileValue.DisplayName`.

File: build/import/Packages.targets [hunk 1]
```diff
@@ -11,7 +11,7 @@
       https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy/nuget/v3/index.json;
       https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json;
       https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json;
-      https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json;
+      https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json
```
Comment (MiYanni) [inline build/import/Packages.targets:14]: Any reason to remove the semi-colon? Just makes it easier to add more sources in the future if it remains.

**Review by MiYanni — APPROVED** [review]

File: build/import/Packages.targets (hunk 1 above)
Comment (tmeschter) [reply build/import/Packages.targets:14]: No, that's just an artifact of changing the sources a couple times.

**Review by drewnoakes — APPROVED** [review]
Please squash this so that we don't have non-compiling commits in the history. I often use git bisect to identify which commit introduced a regression. Having non-compiling commits makes that process painful.

**Review by melytc — APPROVED** [review]

**Comment by drewnoakes** [comment]
@tmeschter is this PR good to go in or are we waiting on something?

**Comment by tmeschter** [comment]
Work on this stalled as I needed to focus on the single-target build for MAUI, but there is otherwise no reason to hold it.

**Pull request merged**